#CREATE request parameters

string endeffector
float32[] position
float32[] orientation 
float32 speed #setPositions
float32 time #positionInterpolationss

--------------------------------
#CREATE response parameters
bool success